csi/moac/crds/mayastorvolume.yaml

```diff
@@ -70,13 +70,16 @@ spec:
                 reason:
                   description: Further explanation of the state if applicable.
                   type: string
-                node:
-                  description: Name of the k8s node with the nexus.
+                publishedOn:
```

Comment:
I know there was a discussion about the name of this. My suggestion would be "location" or "nexusLocation"?

Reply:
Hi Paul, there was actually a discussion about that this morning on slack. The conclusion was to go with `targetNodes` . It is plural because in future we would like to support multiple nexuses for a volume.

Reply:
:+1: